docs/deployment.adoc

```diff
@@ -53,19 +54,21 @@ enabled = true <--.-->
 
 mapping.example.host = {p1-site-slug}.com <--.-->
 mapping.example.source = / <--.-->
-mapping.example.target = /site/default/master/{p1-site-slug}/
+mapping.example.target = /site/default/master/{p1-site-slug}/ <--.-->
 ----
 <.> Enable vhosts configuration
 <.> Intercept requests to `{p1-site-slug}.com`
 <.> Map any request that starts with this path ...
-<.> ... to the internal URL.
+<.> ... to this internal URL.
 
 To read more about virtual hosts, head to the https://developer.enonic.com/docs/xp/stable/deployment/vhosts[_Vhosts_ section of the deployment documentation].
 
 === Task: local vhosts config
 
 To explore how vhosts mappings work, let's configure vhosts for our local XP instance. Then, to test it, we'll configure our local `hosts` file.
 
+NOTE: Because we'll create a mapping to the published version of the site, you should make sure your site is published as described in <<content#publishing, the content chapter>>. Mapping to the draft version of your site would require changing the site's permissions, which isn't covered until the {next-chapter}.
```

Comment:
This is not entirely correct, there are no differences between permissions between draft and master branch - it is the permissions combined with vhost that gives users access to a site. So, you may either map to draft, or just tell people to publish first.

Reply:
Hmm, that may be correct. Does that mean that if you don't explicitly change the site's permissions to allow 'Everyone' read-access, then you will always need an ID provider and a login page?

I tried changing the site's permissions back to the defaults and then publishing again and now I can't seem to access either the draft or the published version without an ID provider.

Reply:
Updated this in defa5c7d938730217a372a18f9318bdc881bd19e along with a few short steps on how to change the permissions of the site, so that the reader doesn't need to think about auth yet.